packages/storybook-vue/stories/3_components/tooltip/Tooltip.stories.mdx

```diff
@@ -0,0 +1,291 @@
+    </template>
+    <scale-button></scale-button>
+    </scale-tooltip>
+  `,
+});
+
+export const Slot2Template = (args, { argTypes }) => ({
+  components: { ScaleTooltip },
+  props: {
+    ...ScaleTooltip.props,
+  },
+  template: `
+    <scale-tooltip
+      :content="content"
+      :placement="placement"
+      :disabled="disabled"
+      :distance="distance"
+      :open="open"
+      :skidding="skidding"
+      :trigger="trigger"
+      :flip="flip"
+      :preventOverflow="preventOverflow"
+    >
+    <template v-slot:content>
+      <scale-rating-stars slot="content" rating="1"></scale-rating-stars>
+    </template>
+    <scale-button></scale-button>
+    </scale-tooltip>
+  `,
+});
+
+export const FocusTemplate = (args, { argTypes }) => ({
+  template: `
+    <div style="padding: 30px 30px 30px 70px">
+    <scale-tooltip trigger="focus" content="Tooltip">
+    <scale-button>Focus me</scale-button>
+    </scale-tooltip>
+    </div>
+  `,
+});
+
+## Standard
+
+<Canvas withSource="none">
+  <Story
+    name="Standard"
+    args={{
+      content: 'Tooltip',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+<ArgsTable story="Standard" />
+
+```html
+<scale-tooltip content="Tooltip">
+  <scale-button>Hover me</scale-button>
+</scale-tooltip>
+```
+
+## Placement
+
+### Left
+
+<Canvas withSource="none">
+  <Story
+    name="Placement-Left"
+    args={{
+      content: 'Tooltip',
+      placement: 'left',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+```html
+<scale-tooltip content="Tooltip" placement="left">
+  <scale-button>Hover me</scale-button>
+</scale-tooltip>
+```
+
+### Right
+
+<Canvas withSource="none">
+  <Story
+    name="Placement-Right"
+    args={{
+      content: 'Tooltip',
+      placement: 'right',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+```html
+<scale-tooltip content="Tooltip" placement="right">
+  <scale-button>Hover me</scale-button>
+</scale-tooltip>
+```
+
+### Below
+
+<Canvas withSource="none">
+  <Story
+    name="Placement-Below"
+    args={{
+      content: 'Tooltip',
+      placement: 'bottom',
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+```html
+<scale-tooltip content="Tooltip" placement="bottom">
+  <scale-button>Hover me</scale-button>
+</scale-tooltip>
+```
+
+## Trigger
+
+<Canvas withSource="none">
+  <Story
+    name="Trigger"
+    args={{
+      content: 'Tooltip',
+      trigger: 'focus',
+    }}
+  >
+    {FocusTemplate.bind({})}
+  </Story>
+</Canvas>
+
+```html
+<scale-tooltip content="Tooltip" trigger="focus">
+  <scale-button>Focus me</scale-button>
+</scale-tooltip>
+```
+
+## Slot
+
+### Link Example
+
+<Canvas withSource="none">
+  <Story name="Slot" args={{}}>
+    {SlotTemplate.bind({})}
+  </Story>
+</Canvas>
+
+```html
+<scale-tooltip>
+  <p>
+    I'm a HTML-tooltip
+    <a slot="content" href="https://github.com/telekom/scale"> with a link</a>
+  </p>
```

Comment:
I think this needs to be:

```html
<scale-tooltip>  
  <a slot="content" href="https://github.com/telekom/scale">I'm a HTML-tooltip with a link</a>
  <scale-button>Hover me</scale-button>
</scale-tooltip>
```

to match the actual template.

Reply:
Oh yes, you are right. Thank you. Changed